_api_app/app/Sites/SitesController.php

```diff
@@ -94,4 +94,19 @@ public function order(Request $request)
         $sites->order($json);
         return response()->json($json);
     }
+
+    public function renderMultisteMenu(Request $request)
+    {
+        $MultisitesMenuRenderService = new MultisitesMenuRenderService(
+            '',
+            true
+        );
+        $res = $MultisitesMenuRenderService->render();
+
+        if ($res === '') {
+            return abort(404, "Not found! lu");
```

Comment:
In case you have only one site you will get 404, because in this case multisite template should return empty string.